### ISSUE TITLE
Conflicting with the docs bundle

### ISSUE THREAD

**Issue by jcldavid** [sparksp/laravel-markdown#3]
When `sparkdown` bundle is loaded together with the `docs` bundle, it's having a conflict with the constants.

Example:

``` Bash
Constant MARKDOWN_VERSION already defined
```

**Comment by stefanneubig** [comment]
+1

**Comment by caneco** [comment]
Do you have any Redirect::home()?

If so… change to Redirect::to('/')

**Comment by sparksp** [comment]
@Caneco I'm not sure how that's relevant to this issue?

**Comment by skcin7** [comment]
I also got this error in Laravel 3.2.14. I solved it by commenting out the extra "MARKDOWN_VERSION" and "MARKDOWNEXTRA_VERSION" constants in bundles/sparkdown/parser.php. For some reason those constants are already being defined in bundles/docs/libraries/markdown.php - I'm not quite sure why the sparkdown library would define them again. Anywho the library works great now... I've been playing around with it and it seems to work great just with this minor hiccup.
